src/odemis/dataio/tiff.py

```diff
@@ -2613,3 +2624,121 @@ def _iterDirectories(tiff_file):
             tiff_file.ReadDirectory()
             dir_index += 1
             yield dir_index
+
+
+ 
+### FEI Metadata parsing
+def _parseINIMetadataAsDict(metadata: str, delimeter: str = "\r\n"):
+    """
+    Parses ini metadata as dictionary
+    metadata (str): metadata string as ini
+    delimeter (str): delimeter between rows (optional, default: "\r\n")
+    return (dict): metadata as dictionary
+    """
+    md = {}
+    for item in metadata.split(delimeter):
+        if item == "":
+            continue # skip blank lines
+        elif re.match(r"\[(.*?)\]", item):
+            subsystem =  item[1:-1] # rm brackets
+            md[subsystem] = {}
+        else:
+            k, v = item.split("=")   
+            md[subsystem][k] = v
+    return md
+
+
+def _convertFEItoOdemisMD(metadata):
+    """
+    Converts FEI metadata to odemis metadata format
```

Comment:
Could you add that this is to parse data from ThermoFisher Scientific SEMs? (I guess "FEI" is going to be less and less known).